mautic/mautic
# Respect default_timezone configuration for dates during segment rebuild

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/7033
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Noticed (https://github.com/mautic/mautic/issues/7033) segments filter doesn't respect default timezone from configuration. This PR added support for it

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com/7097)
2. It's little hard to explain to non tech people.
3. But you have to test on different timezones changed in configuration and check if filters work differently. Also you can check the logs of segment filter queries in System info.
